pkg/node/node.go

```diff
@@ -2,40 +2,37 @@ package node
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
-	"github.com/pkg/errors"
-	"os"
-	"path/filepath"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
 	"k8s.io/klog"
-	"k8s.io/utils/mount"
-
-	"github.com/kubernetes-sigs/container-object-storage-interface-api/apis/objectstorage.k8s.io/v1alpha1"
-	cs "github.com/kubernetes-sigs/container-object-storage-interface-api/clientset/typed/objectstorage.k8s.io/v1alpha1"
 )
 
 var _ csi.NodeServer = &NodeServer{}
 
-const protocolFileName string = `protocolConn.json`
-const credsFileName string = `credentials`
-const fileExistsError string = "File %s already exists"
+const (
+	fileExistsError  = "File %s already exists"
+	credsFileName    = `credentials`
+	protocolFileName = `protocolConn.json`
+)
 
 var getError = func(t, n string, e error) error { return fmt.Errorf("failed to get <%s>%s: %v", t, n, e) }
```

Comment:
can this definition appear in one place?

Reply:
Moved them all into the node.go file